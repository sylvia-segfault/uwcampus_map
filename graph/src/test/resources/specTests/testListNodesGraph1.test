# Create an empty graph
CreateGraph graph1

# Add a vertex into graph1
AddNode graph1 node1

# Returns a list of vertices in graph1
ListNodes graph1