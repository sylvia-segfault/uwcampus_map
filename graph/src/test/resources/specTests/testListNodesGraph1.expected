# Create an empty graph
created graph graph1

# Add a vertex into graph1
added node node1 to graph1

# Returns a list of vertices in graph1
graph1 contains: node1